src/main.js

```diff
@@ -5,8 +5,7 @@ new Vue({
   render: (h) => {
     return h(App, {
       props: {
-        variation: 'resource',
-        locale: 'cs_CZ',
```

Comment:
What's does the change from resource to location entail? And why drop the cs_CZ locale? :)

Reply:
That's only for running it locally, those values have no effect on any production use case. I brought now back the `resource` variant so that we do not change it all the time. Regarding the locale, it's pointless to have any defined language for the local dev env. By default, it takes the language from the browser which is the choice of the dev so it's nice to respect it. It was only a leftover that I didn't want to even push originally 🙈